src/components/SideDrawer/FieldWrapper.tsx

```diff
@@ -126,7 +129,14 @@ export default function FieldWrapper({
           >
             {debugText}
           </Typography>
-
+          <IconButton
+            onClick={() => {
+              copyToClipboard((debugText as string).replace(/\//g, "~2F"));
```

Comment:
```suggestion
              copyToClipboard(debugText as string);
```

Replacing `/` with `~2F` is only required for the Firebase Console URL.
Otherwise, people expect to receive a path delimited with `/`, to work with the Firestore APIs.